README.md

```diff
@@ -30,6 +30,8 @@ specified time, and then repeating the same cycle. The connector always start wi
 
 ### Configuration
 
+// TODO update this section
```

Comment:
I'll update the readme shortly (probably tomorrow).

Reply:
That'd be super, thank you!